app/api/utils.py

```diff
@@ -104,6 +104,12 @@ def get_all_pieces(room_id):
     """
     return sorted(rooms[room_id]['bag'].items(), key=lambda x: x[0])
 
+def not_all_consonants(pieces):
+    """
+Prevents all consonants
+    """
+while not_all_consonants(pieces): pieces = get_pieces(amount, room_id)
```

Comment:
Uhm...I can't seem to find the logic in this function that actually checks if all the pieces are not consonants. It seems to just recurse over itself.

Reply:
The function will recur until conditions are met. I noticed I didn't add the pieces. It is now changed.

Reply:
Sorry, didn't add the changes.